### PR TITLE
Feat [8693w6p7y] add screen reset password

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              /[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]/,
+              password
+            ),
+            padding: 0,
+          }}
+        >
+          <Box
+            component="img"
+            src={passwordHintIcon(
+              /[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]/,
```
Comment (LyubovSugak) [inline src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx:74]: move plz on top and assign to const

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { Box, FormHelperText, List, ListItem, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+const PasswordHintColorAndIcon = ({ password }: { password: string }) => {
+  const { classes } = useStyles()
+
+  const passwordHintColor = (pattern: RegExp, password: string) => {
+    if (password) {
```
Comment (LyubovSugak) [inline src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx:8]: looks a bit like spaghetti code, can't offer another solution right now, but maybe you will be able to do changes. Not critical but though

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            color: passwordHintColor(/[0-9]/, password),
+            padding: 0,
+          }}
+        >
+          <Box component="img" src={passwordHintIcon(/[0-9]/, password)}></Box>
+          &nbsp;1 or more numbers
+        </ListItem>
+        <ListItem
+          sx={{
+            color: passwordHintColor(/[a-zA-Z]/, password),
```
Comment (LyubovSugak) [inline src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx:52]: move regular expression to const on the top

**Review by LyubovSugak — CHANGES_REQUESTED** [review]

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx (hunk 3 above)
Comment (Svitlana-G) [reply src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx]: done

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx (hunk 1 above)
Comment (Svitlana-G) [reply src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx]: done

File: src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx (hunk 2 above)
Comment (Svitlana-G) [reply src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx:10]: Unfortunately, I don't have a better idea now. Maybe something come later

**Review by LyubovSugak — APPROVED** [review]

**Pull request merged**